app/scripts/modules/core/modal/buttons/submitButton.directive.html

```diff
@@ -3,5 +3,5 @@
         ng-click="onClick()">
   <span ng-if="!submitting" class="glyphicon glyphicon-ok-circle"></span>
   <button-busy-indicator ng-if="submitting"></button-busy-indicator>
-  {{label || (isNew ? 'Create' : 'Update')}}
+  {{label || isAdd ? (isNew ? 'Add' : 'Done') : (isNew ? 'Create' : 'Update')}}
```

Comment:
I'd prefer to explicitly pass in the `label` instead of adding more ternary operations to this directive. The one that's in there is bad enough already!